coding-concepts.js

```diff
@@ -40,10 +42,11 @@ const myCodingSkills = {
   databases: "PostgreSQL",
   versionControl: "GitHub"
 }
-// console.log(myCodingSkills.languages[0])
+console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Allllright. I want to say this one is pretty simple and try not to overthink it. "Language" is the "key" and ["JavaScript", "Ruby"] should be the value. Since the code specifies the "0" index and the value itself is an array; initally I think the return would just be "Javascript". But I think it could also be "J" depending on on what's technically counted as the first element of the array? Ultimatley I'm locking in "Javascript". FINAL ANSWER!
+
+// b) Verify and explain: Not gonna lie, four in a row wrong would have hurt! But essentially, as explained above; the code calls on the constant which list multuple keys and values wihtin arrays and strings. But specifically calls on the first key "languages" using dot notation, and again specifys the desired index of "0". In this case, since an array is "0" indexed and the two strings are seperated by a comma. The "0" index would be "Javascript", while the "1" index would be "Ruby".
```

Comment:
Haha, don't worry, it's all about learning. You corrected yourself and explained why you were wrong well, which is all that matters.